### ISSUE TITLE
Missing events description in readme.md

### ISSUE THREAD

**Issue by lgzajac** [smeijer/leaflet-geosearch#101]
First of all - kudos - this module is awesome :)
But it took me some time to find out that I can just subscribe to already existing event :)        
```
this.map.on('geosearch/showlocation', (event) => {
           console.log('on geosearch/showlocation', event);
        });
```
regards!

**Comment by smeijer** [comment]
Thanks for the heads-up.

A pull-request would be most welcome.

**Comment by ville6000** [comment]
I wrote preliminary documentation for events in #117

**Comment by smeijer** [comment]
Closed by #117
